matplotcheck/base.py

```diff
@@ -1026,7 +1033,7 @@ def assert_xlabel_ydata(
         # Testing y-data
         try:
             np.testing.assert_array_max_ulp(
-                np.array(xy_data["y"]), np.array(xy_expected[ycol])
+                np.array(xy_data["y"]), np.array(xy_expected[ycol]), 5
```

Comment:
curious what the 5 is here that was added? @nkorinek when you look at this PR can you please see if it makes sense to you to have 5 there?